Documentation/commands.md

```diff
@@ -72,7 +72,7 @@ In addition to the flags used by individual `rkt` commands, `rkt` has a set of g
 | --- | --- | --- | --- |
 | `--debug` |  `false` | `true` or `false` | Prints out more debug information to `stderr` |
 | `--dir` | `/var/lib/rkt` | A directory path | Path to the `rkt` data directory |
-| `--insecure-options` |  none | **none**: All security features are enabled<br/>**http**: Allow HTTP connections. Be warned that this will send any credentials as clear text.<br/>**image**: Disables verifying image signatures<br/>**tls**: Accept any certificate from the server and any host name in that certificate<br/>**ondisk**: Disables verifying the integrity of the on-disk, rendered image before running. This significantly speeds up start time.<br/>**all**: Disables all security checks | Comma-separated list of security features to disable |
+| `--insecure-options` |  none | **none**: All security features are enabled<br/>**http**: Allow HTTP connections. Be warned that this will send any credentials as clear text.<br/>**image**: Disables verifying image signatures<br/>**tls**: Accept any certificate from the server and any host name in that certificate<br/>**ondisk**: Disables verifying the integrity of the on-disk, rendered image before running. This significantly speeds up start time.<br/>**pubkey**: Allow fetching pubkeys via insecure connections (via HTTP connections or from servers with unverified certificates). The keys will be fetched to a temporary keystore regardless of whether the connection was secure or not.<br/>**all**: Disables all security checks | Comma-separated list of security features to disable |
```

Comment:
> "The keys will be fetched to a temporary keystore"

It is confusing that this flag will do 2 independent things:
- enable download on http
- change the directory where it is downloaded

If users want to change the directory where it is downloaded, couldn't they use `--local-config`?